### PR TITLE
Custom updates

### PR DESCRIPTION
**Pull request by vivekkodira1**

### CODE REVIEW

File: custom/interfaces/installInterfaces.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!bin/bash
+rootFolder=/home/docker/platform/_CUSTOM_/data/contexts/interfaces
+#create the custom folder
+mkdir -p $rootFolder
+#Get the names of interfaces
+folders=`ls -l | grep "^d" | awk '{print $9}'`
+#For each folder
+for folder in $folders; do
+	#copy the definition file in the folder as the interface definition
+	cp -f ./$folder/definition.json $rootFolder/$folder
```
Comment (amithins) [inline custom/interfaces/installInterfaces.sh]: As i understand, interfaces folder will have the definition (with type as the file name) files directly under it.

File: custom/integrations/installIntegrations.sh [hunk 2]
```diff
@@ -1,11 +1,11 @@
 #!bin/bash
-#Get the names of folders
+rootFolder=/home/docker/platform/_CUSTOM_/data/contexts/definitions/integrations
```
Comment (amithins) [inline custom/integrations/installIntegrations.sh]: Is the assumption that **docker** needs to be under **home** is mentioned in the video or docs?

File: custom/interfaces/installInterfaces.sh (hunk 1 above)
Comment (vivekkodira1) [inline custom/interfaces/installInterfaces.sh]: @amithins That is what is occurring here too. This command is copying the file 'googlesheets/definition.json' to the file '../interfaces/googlesheets

File: custom/integrations/installIntegrations.sh (hunk 2 above)
Comment (vivekkodira1) [inline custom/integrations/installIntegrations.sh]: @amith yes. Docs contain this note.

**Pull request merged**